consensus/src/commons.rs

```diff
@@ -261,9 +261,9 @@ impl IterCounter for u8 {
     fn step_from_name(&self, st: StepName) -> Self::Step {
         let sel_num = self * Self::STEP_NUM;
```

Comment:
Can we rename `sel_num` ? I guess `sel` referred to Selection

Reply:
Done